gobblin-utility/src/main/java/gobblin/util/JobConfigurationUtils.java

```diff
@@ -78,10 +82,19 @@ public static void putStateIntoConfiguration(State state, Configuration configur
    * @param fileName the name of the file to load properties from
    * @return a new {@link Properties} instance
    */
-  public static Properties fileToProperties(String fileName) throws IOException, ConfigurationException {
-    Path filePath = new Path(fileName);
+  public static Properties fileToProperties(String fileName)
+      throws IOException, ConfigurationException, URISyntaxException {
+
     PropertiesConfiguration propsConfig = new PropertiesConfiguration();
-    propsConfig.load(filePath.getFileSystem(new Configuration()).open(filePath));
+    Configuration conf = new Configuration();
+    Path filePath = new Path(fileName);
+    URI fileURI = filePath.toUri();
+
+    if (fileURI.getScheme() == null && fileURI.getAuthority() == null) {
```

Comment:
It's the same logic the Hadoop `FileSystem` uses to check whether or not it should use the default scheme: http://grepcode.com/file/repo1.maven.org/maven2/org.apache.hadoop/hadoop-common/2.3.0/org/apache/hadoop/fs/FileSystem.java#356